### DISCUSSION TITLE
File Manager / Browser

### DISCUSSION THREAD

**Discussion by hroma** [odygrd/quill#274] [Q&A]
Hi,

Is there an existing project/plugin to handle uploaded files like a file browser/manager/picker?

I need to migrate a project where tinyMCE3 with [RoxyFileman](https://sourceforge.net/projects/roxyfileman/) is omni present... Last update of RoxyFileman was 10 years ago... I think quill will be the perfect fit for this project but an image selector/uploader is mandatory in this project...

Thank in advance for your reply(s)

**Comment by odygrd** [comment]
hey, quill is a logging library in C++ used to create log files.  It isn't related to managing filesystem files

**Comment by hroma** [comment]
Sorry... wrong library... I though I was writting in Quilljs... you can delete the discussion.
